### PR TITLE
Adding Sagemaker pipeline sample

### PR DESCRIPTION
**Pull request by prayeole**
Issue #, if available:

Description of changes: Adding a Sagemaker pipeline example for creating and executing a pipeline run.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: samples/pipeline/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+### Updating the Pipeline Specification
+
+In the `pipeline.yaml` file, modify the placeholder values with those associated with your account. 
+
+## Submitting your Pipeline Specification
+
+### Modify/Create a JSON pipeline definition
+
+Create the JSON pipeline definition using the JSON schema documented at https://aws-sagemaker-mlops.github.io/sagemaker-model-building-pipeline-definition-JSON-schema/. You will need to convert your JSON pipeline definition into String format to pass to the *.spec.pipelineDefinition* key in the Kubernetes YAML spec. You may use online third-party tools to convert from JSON to String format. In this sample, you are provided a sample pipeline definition with one Training step.
```
Comment (a-hilaly) [inline samples/pipeline/README.md:17]: It is possible to pass a json object as is using the [yaml-multi-line ](https://yaml-multiline.info/)directive. Something similar to https://github.com/aws-controllers-k8s/iam-controller/blob/main/test/e2e/resources/role_simple.yaml#L9-L21 . 

However not sure if the sagemaker API accepts indented json strings

File: samples/pipeline/README.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+### List pipelines and pipeline executions
+
+To list all pipelines created using the ACK controller use the following command:
+```
+$ kubectl get pipeline.sagemaker.services.k8s.aws
+```
+If it is a pipeline executions it is endpointsconfigs.sagemaker.services.k8s.aws  
+```
+$ kubectl get pipelineexecution.sagemaker.services.k8s.aws
+```
```
Comment (a-hilaly) [inline samples/pipeline/README.md:41]: nit: you could also use short names such as `pipeline` and `pipelineexecution`

**Review by a-hilaly — COMMENTED** [review]
This is great, thank you @prayeole ! I left two tiny comments in-line

File: samples/pipeline/README.md (hunk 2 above)
Comment (prayeole) [reply samples/pipeline/README.md]: Updated this as suggested.

**Review by prayeole — COMMENTED** [review]
Updates made. Comments in-line.

**Review by ryansteakley — APPROVED** [review]
Nice thanks for the contribution! Lets see if @suryans-commit  has any comments

**Pull request merged**